src/api/app/controllers/webui/package_controller.rb

```diff
@@ -798,11 +798,11 @@ def live_build_log
     end
 
     begin
-      @package = Package.get_by_project_and_name(params[:project], params[:package],
+      @package = Package.get_by_project_and_name(@project, params[:package],
                                                  use_source: false, follow_project_links: true)
     rescue Package::UnknownObjectError
-      flash[:error] = "Couldn't find package '#{params[:package]}' in project '#{params[:project]}'. Are you sure it exists?"
-      redirect_to project_show_path(@project.name)
+      flash[:error] = "Couldn't find package '#{params[:package]}' in project '#{@project.to_param}'. Are you sure it exists?"
+      redirect_to project_show_path(@project.to_param)
```

Comment:
IIRC path helpers like project_show_path also accept ActiveRecord objects as parameter. So you might be able to shorten this with ```project_show_path(@project)```.